tools/run_tests/sanity/check_bazel_workspace.py

```diff
@@ -74,6 +75,7 @@
     'opencensus_proto',
     'com_envoyproxy_protoc_gen_validate',
     'com_google_googleapis',
+    'com_github_cncf_udpa',
```

Comment:
why is this under BAZEL_ONLY_DEPS?    it seems there's a third_party/udpa submodule so it should be possible to check that the bazel dep and the submodule are at the same commit. Same for other "external proto" libraries.